parser/util.go

```diff
@@ -19,6 +22,30 @@ func ResolvePath(root, path string) (string, error) {
 	return filepath.Join(root, path), nil
 }
 
+// ExpandHomeDir expands the path to include the home directory if the path is
+// prefixed with `~`. If it isn't prefixed with `~`, the path is returned as-is.
+func ExpandHomeDir(path string) (string, error) {
+	if len(path) == 0 {
+		return path, nil
+	}
+
+	if path[0] != '~' {
+		return path, nil
+	}
+
+	if len(path) > 1 && path[1] != '/' && path[1] != '\\' {
```

Comment:
Should this check for `filepath.Separator` specifically?

Reply:
This comes from: https://github.com/mitchellh/go-homedir/blob/master/homedir.go#L55-L58
Which is already cross-platform, didn't want to change the logic too much there.